### DISCUSSION TITLE
More examples per genre and tips and tricks

### DISCUSSION THREAD

**Discussion by timtensor** [Rainbow-Dreamer/musicpy#53] [General]
Hi, first of all really enjoying the library. Secondly i was wondering would it be possible to have more examples shared , perhaps by genre like  for example there are usually common chord progressions for house music, electronic music etc. As a first time user i feel that unique sounds , chord patterns could be generated. Just wondering it could be great if there is a dedicated section for that.

**Comment by Rainbow-Dreamer** [comment]
There are 3 sections of musicpy examples in wiki, each section has around 10 examples for different genres, [here](https://github.com/Rainbow-Dreamer/musicpy/wiki/musicpy-composition-code-examples-part-1) is the first section.

**Comment by timtensor** [comment]
Thank you . I just wanted to give a suggestion like to have different genres grouped. For example if it's jazz beat how can the drum pattern would look like ? Or how can you generate funky short guitar rythms , bass rythms etc ! 

The lib is so exhaustive , it is great . Just I think it a bit overwhelming to a begginer . 

Another thought could be that genre classification would help in perhaps making an AI model in sturctured way out of midi ? Just a thought .

**Comment by Rainbow-Dreamer** [comment]
Thanks for the suggestion, I will improve the examples quality in the future in my spare time.
